<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>207</width>
    <height>119</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Material</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <widget class="QToolButton" name="OKButton">
   <property name="geometry">
    <rect>
     <x>116</x>
     <y>86</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="E_Input">
   <property name="geometry">
    <rect>
     <x>84</x>
     <y>34</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="G_Input">
   <property name="geometry">
    <rect>
     <x>84</x>
     <y>60</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>34</y>
     <width>73</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>E :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="MatID_Input">
   <property name="geometry">
    <rect>
     <x>84</x>
     <y>8</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>8</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Macterial ID:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>25</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>G :</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>MatID_Input</tabstop>
  <tabstop>E_Input</tabstop>
  <tabstop>G_Input</tabstop>
  <tabstop>OKButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
